docs/analyzer/user_guide.md

```diff
@@ -717,6 +717,11 @@ analyzer arguments:
                         clang-tidy checkers. The file can be dumped by
                         'CodeChecker analyzers --dump-config clang-tidy'
                         command.
+  --checker-config [CHECKER_CONFIG [CHECKER_CONFIG ...]]
+                        Checker configuration options in the following format:
+                        analyzer:key=value. The collection of the options can
+                        be printed with 'CodeChecker checkers
```

Comment:
If I run the following command: `CodeChecker checkers --checker-config` I will get the following error message:
```sh
Traceback (most recent call last):
  File "~/codechecker/build/CodeChecker/cc_bin/CodeChecker.py", line 128, in main
    args.func(args)
  File "~/codechecker/build/CodeChecker/cc_bin/../lib/python3/codechecker_analyzer/cmd/checkers.py", line 197, in main
    working)
  File "~/codechecker/build/CodeChecker/lib/python3/codechecker_analyzer/analyzers/analyzer_types.py", line 203, in build_config_handlers
    construct_config_handler(args, context)
  File "~/codechecker/build/CodeChecker/lib/python3/codechecker_analyzer/analyzers/clangtidy/analyzer.py", line 292, in construct_config_handler
    for cfg in args.checker_config:
TypeError: 'bool' object is not iterable
```